controllers/component_build_controller_test.go

```diff
@@ -761,6 +761,108 @@ var _ = Describe("Component initial build controller", func() {
 			deleteRoute(pacRouteKey)
 		})
 
+		It("should fail to unconfigure PaC when pac secret is missing", func() {
```

Comment:
Usually the most common successful use case is the first test.
Also, successful cases goes first, then failure cases. However there are exceptions if the test cases are grouped logically. Please try to follow this convention.

Reply:
ack reordered